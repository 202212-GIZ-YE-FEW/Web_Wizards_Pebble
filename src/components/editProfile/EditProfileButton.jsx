import { Button } from "react-flatifycss";

const EditprofileButton = (props) => {
    const { text, className = "" } = props;
    return (
        <Button
            className={`!w-36 !h-16 !py-2 !px-1 !transition !bg-white !border-2 !border-[black] !rounded-lg !shadow-[2px_2px_#1A1A1A] ${className}`}
        >
            {text}
        </Button>
    );
};

export default EditprofileButton;
