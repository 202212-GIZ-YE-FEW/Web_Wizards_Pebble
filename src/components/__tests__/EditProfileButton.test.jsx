import renderer from "react-test-renderer";

import EditProfileButton from "@/components/editProfile/EditProfileButton";

it("renders correctly", () => {
    const tree = renderer.create(<EditProfileButton />).toJSON();
    expect(tree).toMatchSnapshot();
});
