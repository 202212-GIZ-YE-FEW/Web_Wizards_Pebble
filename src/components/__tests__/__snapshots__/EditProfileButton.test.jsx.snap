// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className=" w-1/3 flex justify-center lg:py-3 lg:px-3 md:py-2 md:px-2 sm:py-1 sm:px-2  "
>
  <button
    className="button__ButtonWrapper-sc-1aw98b4-0 button w-full h-16 px-1 transition "
    disabled={false}
    style={
      Object {
        "backgroundColor": "white",
        "border": "2px solid black",
        "borderRadius": "0.5rem",
        "boxShadow": "2px 2px #1a1a1a",
      }
    }
  >
    <div
      className="relative flex items-center space-x-4 justify-center"
    >
      <span
        className="block w-max font-semibold tracking-wide text-gray-900 sm:text-base text-sm md:text-lg lg:text-xl "
      />
    </div>
  </button>
</div>
`;
